posthog/clickhouse/client/execute_async.py

```diff
@@ -47,11 +45,11 @@ def __init__(self, query_id: str, team_id: int):
 
     @property
     def results_key(self) -> str:
-        return f"{self.KEY_PREFIX_ASYNC_RESULTS}:{self.team_id}:{self.query_id}"
+        return f"{self.KEY_PREFIX_ASYNC_RESULTS}:{self.query_id}:{self.team_id}"
```

Comment:
We can't change this, because then we won't be able to return results of async queries calculated before this change was deployed

Reply:
This should only break queries that are currently executing at deploy time, right?

I can add a little bridge so that it checks the old key if the new key is empty, and then remove it after a few days.

Reply:
Yup, that'd work

Reply:
Awesome - this ugly thing should patch that
https://github.com/PostHog/posthog/pull/22372/commits/5a8d06f62e9ace286a2e22465a0df9e2e256717a